test/build.info

```diff
@@ -260,7 +260,7 @@ IF[{- !$disabled{tests} -}]
 
   INCLUDE[testutil.o]=..
   INCLUDE[ssl_test_ctx.o]=../include
-  INCLUDE[handshake_helper.o]=../include
+  INCLUDE[handshake_helper.o]=../include ../ssl
```

Comment:
Ah, I forget that the PACKET API is all inline.  But still, yeah, slippery slope 'n all that.